tutorials/3d/portals/fundamentals.rst

```diff
@@ -0,0 +1,102 @@
+Fundamentals
+============
+The RoomManager
+~~~~~~~~~~~~~~~
+Anytime you want to use the portal system, you need to include a special node in your scene tree, called the :ref:`RoomManager<class_RoomManager>`. The RoomManager is responsible for the runtime maintenance of the system, especially converting the objects in your rooms into a `room graph` which is used at runtime to perform occlusion culling and other tasks.
+
+Room Conversion
+^^^^^^^^^^^^^^^
+This conversion must take place every time you want to activate the system, it does not store the `room graph` in your project (for flexibility and to save memory). You can either trigger it by pressing the ``Convert Rooms`` button in the editor toolbar when the RoomManager is selected, or you can call the ``rooms_convert`` function in the RoomManager. This latter method will be what you use in game. Note that for safety, best practice is to call ``rooms_clear`` before unloading / changing levels.
+
+If you convert the level while the editor is running, the portal culling system will take over from the normal Godot frustum culling. This may affect some editor features. For this reason, you can turn the portal culling on and off, using the ``Active`` setting in the RoomManager.
```

Comment:
```suggestion
If you convert the level while the editor is running, the portal culling system will take over from the normal Godot frustum culling. This may affect some editor features. For this reason, you can turn the portal culling on and off, using the **Active** property in the RoomManager node.
```